src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,5 @@
+<hibernate-configuration xmlns="http://www.hibernate.org/xsd/hibernate-configuration">
+    <session-factory>
+
+    </session-factory>
+</hibernate-configuration>
```

Comment:
where is your configuration? and better to add  one empty line at the end of the file

Reply:
not addressed